reference-data-proxy/Dockerfile

```diff
@@ -1,11 +1,19 @@
 # Node Alpine Docker Image
-FROM node:17-alpine3.14
+FROM node:14.18.1-alpine3.13
```

Comment:
this should work with v17. does it throw any errors?

Reply:
didn't mean to commit this, i'll revert.